flink/v1.14/flink-runtime/pom.xml

```diff
@@ -213,6 +213,7 @@
                             <include>com.github.ben-manes.caffeine:*</include>
                             <include>org.apache.avro:*</include>
                             <include>org.apache.orc:*</include>
+                            <include>io.airlift:*</include>
```

Comment:
Is this necessary for orc format?

Reply:
> Is this necessary for orc format?

orc is depedended on io.airlift, it should included in some flink environment, to avoid lacking of dependency